lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -185,6 +185,10 @@ export const ClientAuthErrorMessage = {
     bindingKeyNotRemovedError: {
         code: "binding_key_not_removed",
         desc: "Could not remove the credential's binding key from storage."
+    },
+    logoutNotSupported: {
```

Comment:
Is this actually a valid configuration to run in?

Reply:
The authority.  Signout flows are generally considered P0.  Do we support an authority that wouldn't allow users to log out?

Reply:
I've ask the b2c folks for clarification, but this seems to be by design (I'm guessing because ropc doesnt set cookies?).